ui/revenuecatui/src/test/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/TemplateConfigurationFactoryTest.kt

```diff
@@ -87,14 +81,20 @@ internal class TemplateConfigurationFactoryTest {
 
     private fun getPackageInfo(rcPackage: Package): TemplateConfiguration.PackageInfo {
         val localizedConfiguration = TestData.template2.configForLocale(Locale.US)!!
+        val periodName = when(rcPackage.packageType) {
+            PackageType.ANNUAL -> "Annual"
+            PackageType.MONTHLY -> "Monthly"
+            PackageType.WEEKLY -> "Weekly"
+            else -> error("Unknown package type ${rcPackage.packageType}")
```

Comment:
I didn't like copying this here... But I thought it was better to kind of duplicate this logic than use the real code during the tests.